components/automate-chef-io/content/docs/reports.md

```diff
@@ -89,73 +88,82 @@ The download button located to the right of the search bar allows the user to do
 
 Severity of Node Failures
 : Indicates the severity of the compliance scan failures on your system's nodes.
 
-Global Compliance Over Time
-: Represents the changes in size and compliance status of your system over times. The calendar in the upper right corner of the visualization allows you to change the dates represented in this view,
+Node Status Over Time
+: Represents the changes in size and compliance status of your system over times. The dropdown menu in the upper left corner of the visualization allows you to change the date range represented in this view. Modifying the date range will affect the display of the other visualizations.
 
 Top Platform Failures
-: Represents the amount of compliance scan failures ranked by operating system. Hovering over an individual bubble shows the platform and the number of nodes impacted.
+: Represents the amount of compliance scan failures ranked by operating system. Hovering over an individual bubble shows the platform name and the number of impacted nodes.
 
  Top Environment Failures
-: Represents the amount of compliance scan failures ranked by environment. Hovering over an individual bubble shows the environment and the number of nodes impacted.
+: Represents the amount of compliance scan failures ranked by environment. Hovering over an individual bubble shows the environment name and the number of impacted nodes.
 
 ### Profile Status
 
-The profile status overview provides insight into your system's compliance status from the compliance perspective of Compliance profiles run during scans.
+The _Profile Status_ view provides insight into your system's compliance status from the compliance perspective of Compliance profiles run during scans.
 
 Control Status
-: The control status displays the number of controls in your system, organized by passing, failing, and skipped controls.
- 
+: Displays the number of controls in your system, organized by passing, failing, and skipped controls.
+
 Severity of Control Failures
 : Displays a ranked view of the number and significance of failed controls, sorted into critical, major, and minor control failures.
 
-Test Results Over Time
-: A line graph representing the quantity of controls and Compliance scan results over time. The time-frame can be altered using calendar button in the upper right corner of the chart.
+Control Status Over Time
+: A line graph representing the quantity of controls and Compliance scan results over time. Modify the time-frame using the calendar button in the upper right corner of the chart.
 
 Top Profile Failures
-: Indicates the profiles with the highest failure rate. Hovering over an individual bubble shows the full name of the profile and the number of nodes impacted.
+: Indicates the profiles with the highest failure rate. Hovering over an individual bubble shows the full name of the profile and the number of impacted nodes.
```

Comment:
```suggestion
: Indicates the profiles with the highest failure rate. Hover over an individual bubble to show the full profile name and the number of impacted nodes.
```
I like indicates here because the relationship is _indexical_, meaning it points to other things. Are impacted nodes failed nodes, or node with the profile?

Reply:
It refers to failed nodes.
When "profile" is mentioned here, it refers to the operating system, but the graph is labeled "Top Profile Failures."
<img width="299" alt="Chef_Automate" src="https://user-images.githubusercontent.com/5807053/89825578-ce9d2e80-db22-11ea-912d-53f4730b7c16.png">